--- WebProfilerServiceProvider.original.php	2015-03-31 10:12:25.000000000 +0300
+++ WebProfilerServiceProvider.php	2015-03-31 10:12:59.000000000 +0300
@@ -66,6 +66,7 @@
             array('memory',    '@WebProfiler/Collector/memory.html.twig'),
             array('form',      '@WebProfiler/Collector/form.html.twig'),
             array('twig',      '@WebProfiler/Collector/twig.html.twig'),
+            array('ajax',      '@WebProfiler/Collector/ajax.html.twig'), // uxm
         );
 
         $app['data_collectors'] = $app->share(function ($app) {
@@ -78,6 +79,7 @@
                 'time'      => $app->share(function ($app) { return new TimeDataCollector(null, $app['stopwatch']); }),
                 'router'    => $app->share(function ($app) { return new RouterDataCollector(); }),
                 'memory'    => $app->share(function ($app) { return new MemoryDataCollector(); }),
+                'ajax'      => $app->share(function ($app) { return new AjaxDataCollector(); }), // uxm
             );
         });
 
